.babelrc

```diff
@@ -1,3 +1,6 @@
 {
```

Comment:
Duplicate this file, add the plugins required and move it to `/blkn` it may cause incompatibilities.